build-browser.js

```diff
@@ -0,0 +1,23 @@
+const pkg = require('./package.json');
+const rollup = require('rollup').rollup;
+const babel = require('rollup-plugin-babel');
+
+rollup({
+  entry: 'src/index.js',
+  plugins: [
+    babel({
+      presets: ['es2015-rollup'],
+      plugins: ['transform-node-env-inline'],
+      babelrc: false,
+    }),
+  ],
+}).then(bundle => (
+  bundle.write({
+    dest: 'validator.js',
+    format: 'umd',
+    moduleName: pkg.name,
```

Comment:
Does this still work with all of the import schemes defined here? https://github.com/chriso/validator.js/blob/master/validator.js#L24-L34

e.g. https://github.com/chriso/validator.js/pull/448